images/metalk8s-keepalived/keepalived.conf.j2

```diff
@@ -0,0 +1,28 @@
+global_defs {
+    enable_script_security
+}
+
+vrrp_script check_ingress {
+    script "/etc/keepalived/check-ingress.sh"
+    interval 5
+    weight 60
+}
+
+{%- for data in input %}
+
+{%- set is_master = data.get("node") == env["NODE_NAME"] %}
+
+vrrp_instance VI_{{ loop.index }} {
+    state {% if is_master %}MASTER{% else %}BACKUP{% endif %}
```

Comment:
nit (mostly a matter of preference):

```suggestion
    state {{ 'MASTER' if is_master else 'BACKUP' }}
```

Reply:
Yeah it works as well

Reply:
https://github.com/scality/metalk8s/pull/3844/commits/daa0463c09a2fafba3025180159bbccf373473f2